web/gui/layout.css

```
@@ -320,9 +320,78 @@ div.myr_ppi {
+.myr_power_lozenge_name:disabled {
+  cursor: default;
+}
+
+.myr_power_lozenge_name:not(:disabled):hover {
+  background: #333;
+}
+
+.myr_power_lozenge_caret {
+  width: 10px;
+  height: 6px;
+  fill: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)